### DISCUSSION TITLE
4.1.1 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1582] [General]
All devs,

I'm planning to roll a 4.1.1 release. Formally it should contain just fixes. But I want to include some transcription cluster optimizations, like #1566, #1576, #1267. As the versioning rule will be broken, I'll also include #1552, #1556, since they are very minor. Should we include #1553? Any others?

Thanks,
Luis Nassif

**Comment by lfcnassif** [comment]
> Should we include #1553?

I won't include this because of new Strings to be localized, it will be in 4.2.0.

**Reply by lfcnassif** [reply]
> > Should we include #1553?
> 
> I won't include this because of new Strings to be localized, it will be in 4.2.0.

I changed my mind, #1565 depends on it, so I'll include #1553 to avoid resolving conflicts.

**Comment by lfcnassif** [comment]
Current release notes for 4.1.1:
https://github.com/sepinf-inc/IPED/blob/4.1.x/ReleaseNotes.txt#L1-L29

Should we include something else?

**Reply by wladimirleite** [reply]
> Should we include something else?

No, at least from issues that I worked on.
